suite-common/graph/src/graphDataFetching.ts

```diff
@@ -47,6 +51,33 @@ export const addBalanceForAccountMovementHistory = (
     return historyWithBalance;
 };
 
+export const getLatestAccountBalance = async ({
+    coin,
+    descriptor,
+}: {
+    coin: NetworkSymbol;
+    descriptor: string;
+}) => {
+    const networkType = getNetworkType(coin);
+
+    const accountInfo = await TrezorConnect.getAccountInfo({
+        coin,
+        descriptor,
+        suppressBackupWarning: true,
+    });
+
+    if (!accountInfo?.success) {
+        throw new Error(`Get account balance info error: ${accountInfo.payload.error}`);
+    }
+
+    switch (networkType) {
+        case 'ripple':
```

Comment:
comment why is this needed?

Reply:
I don't know either :D but I added comment at least